cluster/log-dump/log-dump.sh

```diff
@@ -502,6 +502,7 @@ function dump_nodes_with_logexporter() {
   local -r cloud_provider="${KUBERNETES_PROVIDER}"
   local -r enable_hollow_node_logs="${ENABLE_HOLLOW_NODE_LOGS:-false}"
   local -r logexport_sleep_seconds="$(( 90 + NUM_NODES / 3 ))"
+  local -r compute_zone="${ZONE:-}"
```

Comment:
ZONE is used in many places. To limit change to regional cluster, lets use a new variable, like LOGEXPORTER_ZONE

Reply:
Done.

Reply:
Hi, as we discussed offline. I don't understand why we need to introduce another zone flag.
IIUC, after this change we will have two sources of ZONE in log-dump.sh script:
* LOGEXPORTER_ZONE used by log-exporter code path
* ZONE used by ssh path (so e.g. if gcs bucket is not set or log exporter failed)

@jkaniuk Why can't we simply use ZONE instead?

Reply:
ZONE is always set. I did not want to change regular behaviour (single zonal clusters) to not introduce some regression. Separate variables allows this change to be applied only to regional clusters.

Reply:
After "offline" discussion with @mborsz - there is some redundancy setting twice the same zone, but changing old behaviour is more risky.
Two ideas solve both issues at the same time:
1. boolean flag, something like `LOGEXPORTER_SINGLE_ZONE`, that would trigger use of that node selector.
2. parameter that passes node selector - both key and value

Reply:
> boolean flag, something like LOGEXPORTER_SINGLE_ZONE, that would trigger use of that node selector.

SGTM.

Reply:
Done.